.github/workflows/publish_docker_images_on_schedule.yml

```diff
@@ -0,0 +1,27 @@
+name: Schedule Build Docker Images For Main Images And Nightly
```

Comment:
Casing

Reply:
Fixed it. I missed this one since it is hidden with the resolved ones on GitHub interface. I need to get used to it 👍